geode-core/src/main/java/org/apache/geode/internal/cache/TXStateProxyImpl.java

```diff
@@ -506,9 +508,26 @@ public Set getAdditionalKeysForIterator(LocalRegion currRgn) {
     return getRealDeal(null, currRgn).getAdditionalKeysForIterator(currRgn);
   }
 
+  /**
+   * A set operation now can bootstrap a transaction now. User need to specifically disable this by
+   * setting this system property to true to get the old behavior.
+   *
+   * @Since Geode 1.3.0
+   */
+  public static final String PREVENT_SET_OP_BOOTSTRAP_TRANSACTION =
```

Comment:
This is a duplicate of the code added to LocalRegion. Is there a way we can make this more DRY? The fact that we have this code in two places suggests to me that LocalRegion is not the right place for it (if it can be placed anywhere else, it probably should).

Reply:
Moved to a SystemPropertyHelper class now. Existing system property and further new ones can be added here for easy access.